spec/models/question_spec.rb

```diff
@@ -50,4 +50,22 @@
       expect(subject.response_choices).to include(subject.question_orders.first.response_choice)
     end
   end
+
+  context 'question renders as sms' do
+    subject { FactoryGirl.create(:question, :with_multiple_response_choices) }
+    expectedString = "hello world?\na) first choice\nb) second choice"
+    
+    it 'with response choices' do
```

Comment:
Perhaps `'renders string correctly'` indicates the behaviour tested better? With usually implies context to me.